matlabCode/createBIDS_anat_Short_T1w_json.m

```diff
@@ -31,60 +35,56 @@
               '_acq-' acq_id ...
               '_run-' run_id '_T1w.json']);
 
+          
+%%
 % Assign the fields in the Matlab structure that can be saved as a json.
-%
-
-%% all REQUIRED metadata fields for Magnetic Resonance Imaging data
-
+% all REQUIRED /RECOMMENDED /OPTIONAL metadata fields for Magnetic Resonance Imaging data
 
 
 
+%% In-Plane Spatial Encoding metadata fields
 
-%In-Plane Spatial Encoding metadata fields
-
-
-%REQUIRED if corresponding fieldmap data is present or when using multiple runs with different phase encoding directions
-%PhaseEncodingDirection is defined as the direction along which phase is was modulated which may result in visible distortions.
+%REQUIRED if corresponding fieldmap data is present or when using multiple 
+% runs with different phase encoding directions phaseEncodingDirection is 
+% defined as the direction along which phase is was modulated which may 
+% result in visible distortions.
 anat_json.PhaseEncodingDirection=' ';
 
-
-
-%REQUIRED if corresponding fieldmap data is present. 
-%The “effective” sampling interval, specified in seconds, between lines in the phase-encoding direction, defined based on the size of the reconstructed image in the phase direction.  
+%REQUIRED if corresponding fieldmap data is present. The effective sampling 
+% interval, specified in seconds, between lines in the phase-encoding direction, 
+% defined based on the size of the reconstructed image in the phase direction.  
 anat_json.EffectiveEchoSpacing=' ';
 
-
-%REQUIRED if corresponding “field/distortion” maps acquired with opposing phase encoding directions are present
-%This is actually the “effective” total readout time , defined as the readout duration, specified in seconds, that would have generated data with the given level of distortion.  It is NOT the actual, physical duration of the readout train
+%REQUIRED if corresponding field/distortion maps acquired with opposing phase 
+% encoding directions are present. This is actually the effective total 
+% readout time , defined as the readout duration, specified in seconds, 
+% that would have generated data with the given level of distortion.  
+% It is NOT the actual, physical duration of the readout train
 anat_json.TotalReadoutTime=' ';
 
 
 
-
-%Timing Parameters metadata fields
+%% Timing Parameters metadata fields
 
 %REQUIRED if corresponding fieldmap data is present or the data comes from a multi echo sequence
 %The echo time (TE) for the acquisition, specified in seconds. 
-%Corresponds to DICOM Tag 0018, 0081 “Echo Time” 
+%Corresponds to DICOM Tag 0018, 0081 “Echo Time�? 
```

Comment:
What are these strange symbols?

Reply:
Not sure. They do not appear on the original file but did show up in this file and plenty others when I opened it in the the matlab IDE after forking and the initial fetching. 
I suspect a weird character encoding "issue" and the double quotes got converted into this: I think that this is due to my git / github desktop set up. If anyone has any clue I am all ears.

Reply:
I think it's because these are 🎇*fancy* 🎇 quotation marks (they're kinda curly and different when opening or closing a quote". 

Can you retype them @Remi-Gau as `"` and see if that fixes it? 

I think it's probably a cross operating system thing that's just weirdly annoying.

Reply:
You are right Kirstie it must be a mix of the original comment having been copy pasted from the BIDS specs and then some character encoding of those fancy quotation marks must have confused my OS.
I have reinserted most of the original double quotes that were around the DICOM tags. I think they look fine now.

Reply:
Awesome!! :tada: